### PR TITLE
Sveltekit example

### PR DESCRIPTION
**Pull request by itsezc**
Adds a Sveltekit example, that is based on the Sveltekit guide on the docs

### CODE REVIEW

**Comment by itsezc** [comment]
Related to https://github.com/moonrepo/moon/issues/543 and https://github.com/moonrepo/moon/pull/544

File: .moon/workspace.yml [hunk 1]
```diff
@@ -1,23 +1,24 @@
 # Trigger CI: 8
 
-$schema: 'https://moonrepo.dev/schemas/workspace.json'
+$schema: "https://moonrepo.dev/schemas/workspace.json"
```
Comment (milesj) [inline .moon/workspace.yml:3]: looks to be ignoring the root prettier settings

File: apps/sveltekit-app/.npmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+engine-strict=true
```
Comment (milesj) [inline apps/sveltekit-app/.npmrc:1]: prob dont need

File: apps/sveltekit-app/package.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "sveltekit-app",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "dev": "vite dev",
+    "build": "vite build",
+    "preview": "vite preview",
+    "test": "playwright test",
+    "check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
```
Comment (milesj) [inline apps/sveltekit-app/package.json:10]: should these be moved to tasks?

Comment (itsezc) [reply apps/sveltekit-app/package.json:10]: Oops, I've created the tasks but missed removing the scripts

**Comment by milesj** [comment]
A lot has changed in this repo, so I went and redid this in another PR.

**Pull request closed without merging**